### PR TITLE
[PotentialFlowApplication] Working again

### PR DESCRIPTION
**Pull request by rubenzorrilla**
With this PR the GUI of the PotentialFlow application is working again. The Kratos solver and processes have been updated accordingly.

Merging in the main repository is still pending but in any case we can merge this now since it is not working in its current state.

By the way, I deactivated the 3D option since it is neither available in Kratos.

### CODE REVIEW

File: kratos.gid/apps/Fluid/write/writeProjectParameters.tcl [hunk 1]
```diff
@@ -3,10 +3,10 @@ proc ::Fluid::write::getParametersDict { } {
     set projectParametersDict [dict create]
 
     # Problem data
-    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict $Fluid::app_id]
```
Comment (rubenzorrilla) [inline kratos.gid/apps/Fluid/write/writeProjectParameters.tcl]: @jginternational everything works after this change but please confirm me that it is OK.

File: kratos.gid/apps/Fluid/write/writeProjectParameters.tcl [hunk 2]
```diff
@@ -3,10 +3,10 @@ proc ::Fluid::write::getParametersDict { } {
     set projectParametersDict [dict create]
 
     # Problem data
-    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict $Fluid::app_id]
+    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict [apps::getActiveAppId]]
 
     # output configuration
-    dict set projectParametersDict output_processes [write::GetDefaultOutputProcessDict $Fluid::app_id]
```
Comment (rubenzorrilla) [inline kratos.gid/apps/Fluid/write/writeProjectParameters.tcl]: Same comment.

File: kratos.gid/apps/Fluid/write/writeProjectParameters.tcl (hunk 1 above)
Comment (jginternational) [reply kratos.gid/apps/Fluid/write/writeProjectParameters.tcl]: This was set to solve the inheritance problem of calling to common functions from the inherited app.

In this case imagine the FSI

the function apps::getActiveAppId will return FSI but you want to execute the function telling the system to search in the Fluid part of the tree, so this is why $Fluid::app_id must be there

**Comment by jginternational** [comment]
Fixes #497

File: kratos.gid/apps/Fluid/write/writeProjectParameters.tcl (hunk 1 above)
Comment (rubenzorrilla) [reply kratos.gid/apps/Fluid/write/writeProjectParameters.tcl]: OK! I'll revert it. Thanks for the explanation.

File: kratos.gid/apps/Fluid/write/writeProjectParameters.tcl (hunk 2 above)
Comment (rubenzorrilla) [reply kratos.gid/apps/Fluid/write/writeProjectParameters.tcl]: (to be reverted as well)

**Comment by rubenzorrilla** [comment]
@jginternational required changes done! Just in case, I tried the FSI and works as expected.

**Comment by jginternational** [comment]
As we say in my town, "chachi pistachi"

**Pull request merged**